app/src/main/java/com/asfoundation/wallet/feature_flags/di/FeatureFlagsModule.kt

```diff
@@ -0,0 +1,67 @@
+package com.asfoundation.wallet.feature_flags.di
+
+import android.content.Context
+import androidx.room.Room
+import cm.aptoide.analytics.AnalyticsManager
+import com.asf.wallet.BuildConfig
+import com.asfoundation.wallet.di.IoDispatcher
+import com.asfoundation.wallet.di.annotations.ShortTimeoutHttpClient
+import com.asfoundation.wallet.feature_flags.FeatureFlagsRepository
+import com.asfoundation.wallet.feature_flags.api.ExperimentsApi
+import com.asfoundation.wallet.feature_flags.db.FeatureFlagsDao
+import com.asfoundation.wallet.feature_flags.db.FeatureFlagsDatabase
+import com.asfoundation.wallet.feature_flags.topup.AndroidIdRepository
+import com.asfoundation.wallet.feature_flags.topup.TopUpDefaultValueProbe
+import com.asfoundation.wallet.feature_flags.topup.TopUpDefaultValueUseCase
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import kotlinx.coroutines.CoroutineDispatcher
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import retrofit2.converter.gson.GsonConverterFactory
+import javax.inject.Singleton
+
+@InstallIn(SingletonComponent::class)
+@Module
+class FeatureFlagsModule {
+
+  @Singleton
+  @Provides
+  fun providesExperimentsApi(
+    @ShortTimeoutHttpClient client: OkHttpClient
+  ): ExperimentsApi = Retrofit.Builder()
+    .baseUrl(BuildConfig.APTOIDE_WEB_SERVICES_AB_TEST_HOST)
+    .client(client)
+    .addConverterFactory(GsonConverterFactory.create())
+    .addCallAdapterFactory(RxJava2CallAdapterFactory.create())
+    .build().create(ExperimentsApi::class.java)
+
+  @Singleton
+  @Provides
+  fun providesFeatureFlagsDatabase(@ApplicationContext context: Context): FeatureFlagsDatabase =
+    Room.databaseBuilder(context, FeatureFlagsDatabase::class.java, "feature_flags_database")
+      .build()
+
+  @Singleton
+  @Provides
+  fun provideFeatureFlagsDao(featureFlagsDatabase: FeatureFlagsDatabase): FeatureFlagsDao =
+    featureFlagsDatabase.getFeatureFlagsDao()
+
+  @Singleton
+  @Provides
+  fun provideTopUpDefaultValueUseCase(
+    featureFlagsRepository: FeatureFlagsRepository,
+    analyticsManager: AnalyticsManager,
+    androidIdRepository: AndroidIdRepository,
+    @IoDispatcher ioDispatcher: CoroutineDispatcher
+  ): TopUpDefaultValueUseCase =
+    TopUpDefaultValueUseCase(
+      TopUpDefaultValueProbe(featureFlagsRepository, analyticsManager),
+      androidIdRepository,
+      ioDispatcher
+    )
```

Comment:
We need, because it needs `TopUpDefaultValueProbe` to be injected for analytics.
And let mb start to organise code into packages for easier separation to modules later.